src/main/java/jp/co/soramitsu/sora/didresolver/config/LoggingFilter.java

```diff
@@ -0,0 +1,62 @@
+package jp.co.soramitsu.sora.didresolver.config;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.security.MessageDigest.getInstance;
+import static org.apache.commons.codec.binary.Hex.encodeHexString;
+import static org.slf4j.MDC.put;
+import static org.slf4j.MDC.remove;
+
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.owasp.security.logging.mdc.MDCFilter;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@WebFilter(filterName = "mdcFilter", urlPatterns = "/*")
+@Slf4j
+public class LoggingFilter extends MDCFilter {
+
+  private static final String MD_5 = "MD5";
+  private static final String REQUEST = "request";
+
+  @Override
+  public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+      throws IOException, ServletException {
+    val req = (HttpServletRequest) request;
+    val serverHost = req.getServerName();
```

Comment:
Why do we need all these local variables?

Reply:
was afraid sonar will warn that methods are called in logger, instead of vars